src/link/filelink.cpp

```diff
@@ -56,6 +62,13 @@ bool FileLink::setConfiguration(const LinkConfiguration& linkConfiguration)
 }
 
 bool FileLink::startConnection() {
+    if(!_file.exists() && QFileInfo(_file).isWritable()) {
```

Comment:
This line (and patch) is not clear to me, can you explain to me more what is going on?

Reply:
This line avoid the creation of new files when startConnection is called.
_needToOpen continues to be true, the file will be created by [this lines](https://github.com/bluerobotics/ping-protocol-cpp/pull/71/commits/751d5d3310192e8e24505d3b5bac764510cf4344#diff-f2c5768500d0f4646a26f6d98be39bd4R30) when a new data is received by the class.
This patch also avoid unnecessary calls of close function, as described by one of the commit.

Reply:
> This line avoid the creation of new files when startConnection is called.

Why and how? Avoids the creation of new files by checking if the file does not exist and is writable? This does not make sense to me.

Reply:
I'll rewrite what I said to make it more clear. 
- If file does not exist and QFile was created with `WriteOnly`or `ReadWrite` (this only happen when creating a log file), the file will be open when new data is received.

Reply:
> the file will be open when new data is received.

This is a behavior of QFile?

Reply:
Yep. You can test it doing:
```
int main()
{
    QFile oi("/oi.txt");
    qDebug() << QFileInfo(oi).isWritable();
    return 0;
}

```